org.bluesabre.MenuLibre.yaml

```diff
@@ -94,7 +94,7 @@ modules:
           - menulibre "$@"
       - type: git
         url: https://github.com/bluesabre/menulibre.git
-        commit: 90ae88e3b90d99ddf2fc5dd1e264081cdc4ddb1d
+        commit: f8c12108324f1aba9a5b17dd1ee24d585672a862
```

Comment:
I don't know if @bluesabre is fine with using this commit. Do you want to use the 2.3.1 tag instead? I didn't use it because it was a prerelease and this one had the appdata fixes.

Reply:
I'd be fine with using the latest 2.3.2 tag, if you don't mind bumping this PR once more. Thanks!

Reply:
Done in 58fac04dc57320296317a6459bd39ef7a81771aa. Thanks